programs/ssl/ssl_server2.c

```diff
@@ -65,7 +65,7 @@ int main( void )
 
 /* Size of memory to be allocated for the heap, when using the library's memory
  * management and MBEDTLS_MEMORY_BUFFER_ALLOC_C is enabled. */
-#define MEMORY_HEAP_SIZE        120000
+#define MEMORY_HEAP_SIZE        180000
```

Comment:
How was this value determined? Is it now sufficient for all certificates?

Reply:
The original memory heap size was enough to handle all the original certificates. I have added a new extension (defined by the standard) that increased the sizes of a few of them. I empirically increased the heap size with 50% and it was more than enough to handle all. To optimize things it can be decreased, but this is a safe value with the current ones. Unfortunately I have no deep knowledge in this part of the library so if we want to determine another value I will need help.

Reply:
"Can we save RAM by not loading PEM certificates in the tests?"

What do you mean by this?

Reply:
Loading a PEM certificates costs the size of the certificate during parsing. But I don't know if it makes a difference to the peak heap usage, or if it's just temporary during parsing and then other things use more space.